sys/etc/fstab

```diff
@@ -0,0 +1,3 @@
+## The root and efi partition is automatically mounted by systemd.
```

Comment:
This doesn't allow you to configure the filesystem options at mount though. Also note this is independent of using systemd-boot (this is based on using systemd in initramfs instead).

Reply:
Got it.